models/components/positional_encoding.py

```diff
@@ -44,16 +45,44 @@ def forward(self, x):
         """
         return x
 
+class SinCosPosEncoding(
```

Comment:
lol this doesn't belong on this pr but shhh

Reply:
yea.. I find it so too - might just be better to separate it lah

Reply:
aiya tai mafan